tests/valid_login.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('login with valid credential', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
+    await page.locator('[data-test="email"]').fill('customer@practicesoftwaretesting.com')
+    await page.locator('[data-test="password"]').fill('welcome01');
+    await page.locator('[data-test="login-submit"]').click();
+
+    await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
```

Comment:
After adding `baseURL` to config, you will not have to specify full URL here as well. Instead, just use RegExp of your specific path to the page: 
```suggestion
    await expect(page).toHaveURL(/\/account$/);
```